config/consts.go

```diff
@@ -29,6 +29,9 @@ const (
 
 	KeyHTTPServerBind = "http.bind"
 	KeyHTTPServerPort = "http.port"
+
+	KeyNomadUrl  = "nomad.url"
+	KeyNomadPort = "nomad.port"
```

Comment:
This is an excellent idea, but we might be missing rest of the implementation. I think, we would need to use something like `viper.GetString(KeyNomadURL)`. I see that at the moment we leverage environment variable, as per (for example):

https://github.com/joyent/triton-service-groups/blob/34497b0cb38024c9fcfe511d874cb62cc7eb911c/groups/orchestrator.go#L127-L128

Would it work with referencing configuration there? There might be some Cobra/Viper magic I am not familiar with there, and if so then I apologise for the confusion.

Reply:
I wasn't sure if this was something we would do later, after this PR, or not.

We'd need to import this value into the configuration struct (`config/config.go`) which is then made available throughout the app. We'd probably create a new `config.Nomad` struct for just these values as well as setup widely used defaults.